README.md

```diff
@@ -90,14 +90,14 @@ The goal of this library is to provide an easy to use interface of Bolt Cloud AP
     response = mybolt.serialRead('10')
     ```
  ##  Utility Functions
-1. isAlive Command (to check the status of device)
+1. isOnline Command (to check recent status of device with time when status updated)
```

Comment:
@Deepanshu625 Add a better description. Something that tells the user exactly what the isOnline command gives.
Eg. The command tells the user about the status of the device if it is Online/Offline along with the last seen time.

Reply:
@sashoeb done.